acceptance_tests/features/environment.py

```diff
@@ -10,4 +14,29 @@ def after_all(context):
 
 
 def before_all(context):
-        database_controller.reset_database('resources/database/databasereset.sql')
+    database_controller.reset_rm_database('resources/database/database_reset_rm.sql')
```

Comment:
`reset_rm_database()` returns `response.text` but it's not used here. Does it need to be handled here at all?

Reply:
When the tests are started this function just resets the database so doesn't need a response.text but this function is also being used to run a number of other scripts through the rm databases that need a response. It should really be renamed as it is being used for more than a reset of the rm database.